src/components/Navbar/Navbar.component.jsx

```diff
@@ -30,8 +31,18 @@ export default function Navbar({ setDarkMode, darkMode }) {
             <MenuIcon />
           </IconButton>
           <SearchBox>
-            <InputBase className={classes.inputSearch} placeholder="Search ..." />
-            <IconButton type="submit" className={classes.iconButton} aria-label="search">
+            <InputBase
+              className={classes.inputSearch}
+              placeholder="Search ..."
+              value={searchParam}
+              onChange={(e) => setSearchParam(e.target.value)}
+            />
+            <IconButton
+              type="submit"
+              className={classes.iconButton}
+              aria-label="search"
+              onClick={() => setParam(searchParam)}
+            >
```

Comment:
I like this search bar, it took me a while to figure out that a click on the search icon was required to run the search 😆